### PR TITLE
(PUP-8122) Change sentence fragment to This depends on ...

### PR DESCRIPTION
**Pull request by ericldelaney**
Change the sentence fragment for listing the dependency cycle errors
from "which depends on" to "This depends on"

### CODE REVIEW

**Comment by ericldelaney** [comment]
jenkins please test this

**Comment by ericldelaney** [comment]
jenkins please test this

**Comment by ericldelaney** [comment]
This passed through ad-hoc

**Comment by ericldelaney** [comment]
jenkins please test this

**Comment by ericldelaney** [comment]
jenkins please test this

File: lib/puppet/module_tool/errors/shared.rb [hunk 1]
```diff
@@ -98,11 +98,11 @@ def initialize(options)
     def multiline
       dependency_list = []
       dependency_list << _("You specified '%{name}' (%{version})") % { name: @source.first[:name], version: v(@requested_version) }
```
Comment (Magisus) [inline lib/puppet/module_tool/errors/shared.rb]: I think this needs a period at the end, with this rephrasing.

**Comment by ericldelaney** [comment]
@Magisus Updated

**Comment by Magisus** [comment]
I was actually referring to the sentence that comes before the one actually changed in the PR (though I agree, they could _both_ use periods).

**Pull request merged**